### ISSUE TITLE
How to change the value of a property in a database row?

### ISSUE THREAD

**Issue by ooomytech** [mariosimao/notion-sdk-php#104]
Hello, tell me what is the correct way to change the value of a property in a row from a database table? And yet, you can show an example of how to add new rows to a table with a data base.

There are no examples at all of how to change property values in pages. Or I didn't find it.

Thanks a lot. I hope you have some time to answer this question.

And thanks a lot for this SDK.

**Comment by mariosimao** [comment]
Sorry for the delay, I have been quite busy lately.

First of all, it is necessary to understand that database rows on Notion are essentially pages where the parent is the database.

### Add a row to a database

```php
<?php

require "vendor/autoload.php";

$token = "your_secret_here";
$notion = \Notion\Notion::create($token);

$databaseId = "your_database_id_here";
$parent = \Notion\Pages\PageParent::database($databaseId);

$title    = \Notion\Pages\Properties\Title::create("Superbad");
$release  = \Notion\Pages\Properties\Date::create(new DateTimeImmutable("2007-10-19"));
$category = \Notion\Pages\Properties\Select::fromName("Comedy")
    ->withColor(\Notion\Pages\Properties\Option::COLOR_BROWN);

$page = \Notion\Pages\Page::create($parent)
    ->withAddedProperty("Title", $title)
    ->withAddedProperty("Release date", $release)
    ->withAddedProperty("Category", $category);

$notion->pages()->create($page);
```

Before:
![image](https://user-images.githubusercontent.com/24276493/185242136-57b5f3d7-4327-43ca-80ef-a46f7d7ced0d.png)

After:
![image](https://user-images.githubusercontent.com/24276493/185242190-3b07838b-397e-4e9d-ae02-7f3123a4fe22.png)

### Change the value of a property

```php
<?php

require "vendor/autoload.php";

$token = "your_secret_here";
$notion = \Notion\Notion::create($token);

$pageId = "your_page_id_here";
$page = $notion->pages()->find($pageId);

$updatedRelease = \Notion\Pages\Properties\Date::create(new DateTimeImmutable("2008-11-04"));
$updatedPage = $page->withAddedProperty("Release date", $updatedRelease);

$notion->pages()->update($updatedPage);
```

Before:
![image](https://user-images.githubusercontent.com/24276493/185242190-3b07838b-397e-4e9d-ae02-7f3123a4fe22.png)

After:
![image](https://user-images.githubusercontent.com/24276493/185242979-582f3b20-583e-415c-9a17-851b1dde4929.png)


Since there are no examples of adding rows to a database and updating properties, I will leave this issue open.

Thanks for your feedback!

**Comment by mariosimao** [comment]
Documentation is available on v0.6.2

- https://mario.engineering/notion-sdk-php/how-to/add-row-to-database.html
- https://mario.engineering/notion-sdk-php/how-to/update-a-page.html#update-properties
